packages/react-html/src/components/tests/Preconnect.test.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+
+import Link from '../Link';
+import Preconnect from '../Preconnect';
+
+describe('<Preconnect />', () => {
+  it('renders a <Link /> with DNS prefetch and preconnect props', () => {
```

Comment:
`renders a <Link /> with a rel prop set to DNS prefetch and preconnect` is more accurate IMO.